packages/client/lib/service/txpool.ts

```diff
@@ -1,18 +1,25 @@
 import Heap from 'qheap'
 import {
+  AccessListEIP2930Transaction,
   Capability,
   FeeMarketEIP1559Transaction,
   Transaction,
   TypedTransaction,
 } from '@ethereumjs/tx'
-import { Address, BN } from 'ethereumjs-util'
+import { Address, BN, bufferToHex } from 'ethereumjs-util'
 import { Config } from '../config'
 import { Peer } from '../net/peer'
 import type VM from '@ethereumjs/vm'
 import type { FullEthereumService } from './fullethereumservice'
 import type { PeerPool } from '../net/peerpool'
 import type { Block } from '@ethereumjs/block'
 
+const MIN_GAS_PRICE_BUMP_PERCENT = 10
+const TX_MAX_DATA_SIZE = 128 * 1024 // 128KB
+const MAX_POOL_SIZE = 5000
+const MIN_GASPRICE = new BN(1000000000) // 1 GWei
```

Comment:
nit, the first constant is `MIN_GAS_PRICE_BUMP_PERCENT` so this one should also have a separator between GAS_PRICE so `MIN_GAS_PRICE` (can also bring it up to the top of the list)